### PR TITLE
Note the default -i option

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -626,6 +626,7 @@ export TIUP_NATIVE_SSH=enable
 > **注意：**
 >
 > 在部署集群的步骤中，若需要使用密码的方式连接 (-p)，或者密钥文件设置了 passphrase，则需要保证中控机上安装了 sshpass，否则连接时会报错。
+> tiup-cluster 的命令行参数 -i 有一个默认值，如果什么都不传的话会使用该默认值，最终 ssh 会接收到 -i <default-value>，如果这不符合预期，可以给 tiup-cluster 传 `-i ""` 覆盖默认值。
```
Comment (lonng) [inline tiup/tiup-cluster.md:629]: 能不能 `--native-ssh` 为 true 的时候，我们自动就把这个默认值设置为空，目前这个用法易用性不好，用户大概率记不住。

Comment (lucklove) [reply tiup/tiup-cluster.md:629]: 感觉可以，那先等一下合并这个文档，我改一下代码

Comment (lucklove) [reply tiup/tiup-cluster.md:629]: -i 的默认值不符合预期是因为 native ssh 可能使用插件，而不是 -i

Comment (lonng) [reply tiup/tiup-cluster.md:629]: OK，那个 PR 改了之后，这里文档应该需要调整

Comment (lucklove) [reply tiup/tiup-cluster.md]: 已调整

**Review by lonng — APPROVED** [review]
LGTM

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -626,6 +626,7 @@ export TIUP_NATIVE_SSH=enable
 > **注意：**
 >
 > 在部署集群的步骤中，若需要使用密码的方式连接 (-p)，或者密钥文件设置了 passphrase，则需要保证中控机上安装了 sshpass，否则连接时会报错。
+> tiup-cluster 的命令行参数 -i 有一个默认值，如果什么都不传的话会使用该默认值，但当使用 `--native-ssh` 且不手动指定 -i 的情况下，tiup-cluster 会将该值置空来使用 ssh 客户端的默认行为。
```
Comment (TomShawn) [inline tiup/tiup-cluster.md:629]: + `会将该值置空`中的`该值`是什么值？
```suggestion
> `tiup-cluster` 的命令行参数 `-i` 有一个默认值。如果没有值传入该参数的话会使用该默认值。但当使用 `--native-ssh` 且不手动指定 `-i` 的情况下，`tiup-cluster` 会将该值置空来使用 SSH 客户端的默认行为。
```

**Comment by lucklove** [comment]
I'm sorry that this PR is out of date, we don't need it any more.

**Pull request closed without merging**